--- a/arch/arm64/boot/dts/qcom/ipq6018.dtsi
+++ b/arch/arm64/boot/dts/qcom/ipq6018.dtsi
@@ -229,15 +229,17 @@
 			no-map;
 		};
 
-		q6_etr_region: q6_etr_dump@50000000 {
-			reg = <0x0 0x50000000 0x0 0x100000>;
-		  	no-map;
-    		};
-
 		m3_dump_region: m3_dump@50100000 {
 			reg = <0x0 0x50100000 0x0 0x100000>;
 		  	no-map;
     		};
+
+		ramoops_region: ramoops@50000000 {
+			compatible = "ramoops";
+			reg = <0x0 0x50000000 0x0 0x100000>;
+			no-map;
+			record-size = <0x1000>;
+		};
 	};
 
 	smem {
